src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -136,6 +137,15 @@ def go_from_trigger_mozilla_central(self):
         r.raise_for_status()
         push_id = r.json()['pushid']
 
+        for zf in self.artifactsHandler.get():
+            with zipfile.ZipFile(zf, 'r') as uz:
+                for i in uz.namelist():
+                    with uz.open(i, 'r') as fl:
+                        entries = fl.read().decode('utf-8').splitlines()
```

Comment:
Isn't reading the entire file ahead of time unnecessary? You should be able to do `[line[3:] for line in fl if line.startswith('SF:')]`